assets/customer.css

```diff
@@ -221,7 +221,9 @@
   }
 
   .customer td::before {
+    color: rgba(var(--color-foreground), 0.75);
     content: attr(data-label);
+    font-size: 1.4rem;
```

Comment:
I am not sure if it changed where it was supposed to because I see it still at 1.6 on mobile ([Visual reference](https://screenshot.click/21-07-17gr0-cedvi.png))

> The order labels on mobile should reduce in size from 1.6rem to 1.4rem.